### PR TITLE
Add script for parsing metrics from CA e2e tests

### PR DESCRIPTION
**Pull request by bskiba**

### CODE REVIEW

File: hack/scripts/ca_metrics_parser.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  Merics are stored in a map keyed by function name and are expressed in 
+  seconds. They include
+  * sum of all samples
+  * count of sumples
+  * average value of samples
+  * upper bound - all collected samples were smaller than this value
+  * buckets - list of tuples (# of samples, bucket upper bound)
+  '''
+  summary = {}
+  with open(sys.argv[1]) as metrics_file:
```
Comment (MaciekPytel) [inline hack/scripts/ca_metrics_parser.py]: shouldn't this use the parameter passed to this function?

File: hack/scripts/ca_metrics_parser.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    summary = {}
+    metrics = json.load(metrics_file)
+    ca_metrics = metrics['ClusterAutoscalerMetrics']
+
+    sum = ca_metrics['cluster_autoscaler_function_duration_seconds_sum']
```
Comment (MaciekPytel) [inline hack/scripts/ca_metrics_parser.py]: sum is a python builtin, can you rename variable?

File: hack/scripts/ca_metrics_parser.py [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+This script parses metrics from Cluster Autoscaler e2e tests.
+'''
+from __future__ import division
+from __future__ import print_function
+import argparse
+import json
+import sys
+
+
+class CAMetric:
```
Comment (MaciekPytel) [inline hack/scripts/ca_metrics_parser.py]: I know it doesn't matter here, but old-style classes are evil and should be eradicated. Therefore 
`class CAMetric(object):`

Comment (bskiba) [inline hack/scripts/ca_metrics_parser.py]: Done.

File: hack/scripts/ca_metrics_parser.py (hunk 2 above)
Comment (bskiba) [inline hack/scripts/ca_metrics_parser.py]: Done.

File: hack/scripts/ca_metrics_parser.py (hunk 1 above)
Comment (bskiba) [inline hack/scripts/ca_metrics_parser.py]: True, done.

**Pull request merged**